exchange-scrapers/config/exchange-scrapers.json

```diff
@@ -124,6 +124,11 @@
       "Symbol": "ZEC",
       "ForeignName": "ZECUSD",
       "Exchange": "Bitfinex"
+    },
+    {
+      "Symbol": "ETH",
+      "ForeignName" : "ETHBTC",
+      "Exchange" : "HitBTC"
     }
```

Comment:
Could you please include the complete list of their pairs?

Reply:
yes, i can add the complete list of the crypto pairs.I'm going to create a new commit in this PR.

Reply:
Hi @frranck,
I have just checked on HitBTC and there more than 500 pairs.Would you like that i include all pairs ? Thanks

Reply:
could you please include the top 30 market caps?